### ISSUE TITLE
Marble noise Left Shoulder -- iCubNancy01

### ISSUE THREAD

**Issue by lpenco** [robotology/icub-support#757]
### Description of the failure
One ball bearing of the left shoulder of the iCub makes some noise when moving the left arm.
Shall we order the piece and change it?

### Detailed conditions and logs of the failure
[ballbearingleftshoulder.MOV.zip](https://github.com/robotology/icub-support/files/2740895/ballbearingleftshoulder.MOV.zip)

**Comment by julijenv** [comment]
Hi @lpenco ,
The noise is probably coming from one of the ball-bearing. The thing is that I can't be sure of the one it is and furthermore, will you be able to change it/them.

To me, we are not talking about a hard job but the problem is that you need to open this motor and for that here is the list of things to do to perform this replacement:

- unwiring the shoulder cable 1,2,5,6,7 and 8
- remove the block rotor and make the change of this/those ball-bearing/s without counting on the fact that the external ring of the little ball bearing usually got broken inside the rotor and you need to extract 
it with a dedicated tool or with a lot of care and patience)
here is a picture of the pack to extract and the 2 ball bearings
![image](https://user-images.githubusercontent.com/7140395/50956511-27769200-14bc-11e9-9afc-1e5546024c0f.png)
![image](https://user-images.githubusercontent.com/7140395/50956861-137f6000-14bd-11e9-84c1-20499b014deb.png)
- reput back on place everything
- **redo the calibration** this part could be tricky

So to sum up, if you feel confident to be able to replace this/those noisy piece(s), consider asking for a  quote for :

> IITCODE | Alias | Qta | type | Description
   1806 | 618-8 | 1 | Bearings (ball bearings, linear guides, balls screws, etc.) | Micro radial ball bearing 8x16x4mm
1803 | 618-4 |  1 | Bearings (ball bearings, linear guides, balls screws, etc.) | Rad. ball bearing 1Row. 4X9X3 - SKF CODE 618/4

Let us know
